### PR TITLE
feat : silicing layout & integration popup add ikp

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Related
[membuat halaman buat ikp baru](https://app.clickup.com/t/9003239225/CES-3438l)

## Changes
- slicing & integration popup ikp
- add props for open followup dialog
- add function for popup add ikp & props for show popup add ikp
- add component base information
- remove props placeholder on base component input text
- add datepicker border when error on style css
- change mandatory validation numeric on vee-validate

## Preview
https://github.com/jabardigitalservice/super-app-cms/assets/44119106/5324c1aa-c5de-4b69-b695-5acce925e899

## Evidence
title: silicing layout & integration popup add ikp
project: Jabar Super Apps
participants: @marsellavaleria19 @yoslie @Ibwedagama @agunghide

### CODE REVIEW

**Review by yoslie — APPROVED** [review]

File: components/Aduan/Dialog/CreateIkp/index.vue [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+      )
+      this.listDataDisposition = responseDisposition.data.data
+    } catch {
+      this.listDataDisposition = []
+    }
+  },
+  computed: {
+    characterTextArea () {
+      return { maxLength: 255, currentLength: this.payload.description.length }
+    },
```
Comment (Ibwedagama) [inline components/Aduan/Dialog/CreateIkp/index.vue:168]: hello @marsellavaleria19 is this computed used?

File: components/Base/Dialog/Footer/index.vue [hunk 2]
```diff
@@ -9,10 +9,11 @@
           <jds-button
             v-show="showCancelButton"
             :label="labelButtonCancel"
+            type="button"
             variant="secondary"
             class="!text-[14px] !font-bold"
             :disabled="isDisabledButtonCancel"
-            @click="$emit('close')"
+            @click="closePopup"
```
Comment (Ibwedagama) [inline components/Base/Dialog/Footer/index.vue:16]: hello @marsellavaleria19 I think we don't have to create a method for this. can't we use `@click.prevent="$emit('close')"` instead?

**Review by Ibwedagama — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: components/Aduan/Dialog/CreateIkp/index.vue (hunk 1 above)
Comment (marsellavaleria19) [reply components/Aduan/Dialog/CreateIkp/index.vue]: hello @Ibwedagama.. No, I'm not used that computed.. I forgot to delete that computed.. I've fixed it, this is my commit : https://github.com/jabardigitalservice/super-app-cms/pull/177/commits/39e6dbe55493eea3ca07da84ffe9b4232afb79e3

File: components/Base/Dialog/Footer/index.vue (hunk 2 above)
Comment (marsellavaleria19) [reply components/Base/Dialog/Footer/index.vue]: hello @Ibwedagama , yes we can used that.. Thank for advice.. I've fixed it, this is my commit :   https://github.com/jabardigitalservice/super-app-cms/pull/177/commits/39e6dbe55493eea3ca07da84ffe9b4232afb79e3

**Pull request merged**